### ISSUE TITLE
Add Sendinblue - Terms of Service

### ISSUE THREAD

**Issue by martinratinaud** [OpenTermsArchive/contrib-declarations#52]
New service addition requested through the contribution tool

You can see the work done by the awesome contributor here:
https://opentermsarchive.org/fr/contribute/service?documentType=Terms%20of%20Service&name=Sendinblue&selectedCss[]=main&step=2&url=https%3A%2F%2Ffr.sendinblue.com%2Flegal%2Ftermsofuse%2F&expertMode=true

Or you can see the JSON generated here:
```json
{
  "name": "Sendinblue",
  "documents": {
    "Terms of Service": {
      "fetch": "https://fr.sendinblue.com/legal/termsofuse/",
      "select": [
        "main"
      ]
    }
  }
}
```

You will need to create the following file in the root of the project: `services/Sendinblue.json`

Transfered from https://github.com/ambanum/OpenTermsArchive/issues/690

**Comment by Kissaki** [comment]
This ticket is a duplicate of #3 and was implemented in #761. It should consequently be closed.

**Comment by MattiSG** [comment]
Thanks @Kissaki!
